etc/testing/circle/deploy-pachyderm.sh

```diff
@@ -3,9 +3,12 @@
 set -ex
 
 export PATH=$(pwd):$(pwd)/cached-deps:$GOPATH/bin:$PATH
+export PACHYDERM_VERSION="$(jq -r .pachyderm version.json)"
+
+helm repo add pachyderm https://pachyderm.github.io/helmchart
+helm repo update
+helm install pachd pachyderm/pachyderm --set deployTarget=LOCAL --version ${PACHYDERM_VERSION}
 
-echo 'y' | pachctl deploy local
 kubectl wait --for=condition=available deployment -l app=pachd --timeout=5m
 pachctl version
-
-pachctl config update context "$(pachctl config get active-context)" --pachd-address="$(minikube ip):30650"
+rm -f ~/.pachyderm/config.json
```

Comment:
Does this file persist in CircleCI across jobs?

Reply:
Updating the `pachd-address` here is what was causing those tests to fail so figured I'd get rid of that file in case it does exist. Just pushed another commit getting rid of the `rm -f ~/.pachyderm/config.json` as it's not needed.